recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "laspy" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "9561d6d4cfb5ebfa8201151415d45085" %}
```

Comment:
Thanks for the help @synapticarbors . May I ask where did you get this sha256 hash??? I checked the md5 links on https://pypi.python.org/pypi/laspy#downloads but none of them shows that number.